posts/2015-04-20-mix-it-2015-pt1.md

```diff
@@ -0,0 +1,50 @@
+
+- deux séances de Live Coding consacrées à des librairies Frontend : [les ficelles de Marionnette](http://www.mix-it.fr/session/801/apprenez-les-ficelles-de-marionette), un plugin Backbone.js présenté de manière très vivante par Romain Maton, qui a pris le temps d'expliquer les principes de Backbone au public majoritairement ignorant de cette technologie, et [React.js pour les néophytes](http://www.mix-it.fr/session/1991/reactjs-pour-les-neophytes), séance durant laquelle Nicolas Cuillery a développé devant l'assistance une application mettant en oeuvre les différents composants concourant au pattern [Flux](https://facebook.github.io/flux/docs/overview.html) (Store, Dispatcher & View) à l'aide de la librairie JS qui a actuellement le vent en poupe : React.js.
+
+
+##Bases de données
+
+Au cours de sa conférence intitulée [Tips and tricks for clean relational db schemas](http://www.mix-it.fr/session/1241/tips-and-tricks-for-clean-relational-db-schemas), Clément Delafargue a insisté sur la nécessité de reprendre la main sur son schéma et cesser de déléguer cette tâche aux ORM, qu'il semble ne pas particulièrement apprécier et auxquels il préfère des librairies plus légères et davantage orientées requêtes ([POMM](http://www.pomm-project.org/), [Jooq](http://www.jooq.org/learn/), etc.). Il a également fait quelques rappels de base sur la conception d'un bon schéma de données et mentionné quelques fonctionnalités très utiles proposées par [PostgreSQL](http://www.postgresql.org/), un système de gestion de base de données qu'il affectionne (et nous ne saurions le blâmer pour ce choix). En résumé, des conseils très utiles dispensés par un conférencier qui assume manifestement certains partis pris.
+
+On ne quitte pas le domaine des données, avec la [conférence](http://www.mix-it.fr/session/1511/orientdb-the-2nd-generation-of-multimodel-nosql) au cours de laquelle Luigi Dell'aquila a présenté la base de données No-SQL orientée graphes [OrientDB](http://orientdb.com/orientdb/). OrientDB se pose en tant que base de données multi-modèles puisqu'elle permet de modéliser des données dans la plupart des formats No-SQL (clés/valeurs, colonnes, graphes, documents). La liste de ses fonctionnalités s'avère particulièrement impressionnante, jugez plutôt : requêtage SQL possible, schéma de données optionnel, transactions, sharding, gestion des groupes et permissions, etc. Elle semble combiner les avantages des deux mondes (SQL et No-SQL) et notamment pallier les limitations du modèle relationnel sur de gros volumes. Luigi Dell'aquila est un orateur remarquablement plaisant et enthousiaste, qui sait capter l'attention de l'auditeur. Sa conférence donne sincèrement envie de s'intéresser à l'outil. Reste à confirmer, à l'usage, les espoirs nés des promesses d'OrientDB.
```

Comment:
It could be nice to add a link to http://en.wikipedia.org/wiki/Shard_%28database_architecture%29 on the word `sharding`.